pages/news-and-events/community-spotlight/success-stories/_id.vue

```diff
@@ -0,0 +1,333 @@
+<template>
```

Comment:
Wireframes show a link to the success stories page under the details: 
![image](https://user-images.githubusercontent.com/5529126/124760240-8935dc80-defe-11eb-973a-e736405fd782.png)

Reply:
Thanks I missed that